src/.nuget/packages.config

```diff
@@ -3,4 +3,5 @@
   <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00040" />
   <package id="dnx-coreclr-win-x86" version="1.0.0-beta5-11580" />
   <package id="Microsoft.DotNet.BuildTools.ApiTools" version="1.0.0-prerelease" />
+  <package id="dnx-mono" version="1.0.0-beta5-11596" />
```

Comment:
I'm not sure when nuget team plans to deliver a cross-plat nuget.exe that supports v3 restore.  It's not in the super near future AFAIK so we'll continue to use DNX for at least the next few weeks.  I'd split it.